composer.json

```
@@ -17,6 +17,9 @@
   "require": {
       "composer/installers": "*"
   },
+  "replace": {
+      "twbs/bootstrap": "*"
```

Question: would it be better to specify that this package only replaces a specific `twbs/bootstrap` release branch?
```suggestion
      "twbs/bootstrap": "^4"
```

---

I think so - it demonstrates the intent more clearly and it might make some composer aspects more performant, too.

---

We should update this as well https://github.com/az-digital/arizona-bootstrap-packagist/blob/main/.github/workflows/release.yml

So that it knows to switch versions when a 5.x release is made.

---

> Would this even work at all without pinning to v4 of twbs for now?

@mmunro-ltrr I'm thinking no (part of why I asked the question in the first place).

---

> We should update this as well [`main`/.github/workflows/release.yml](https://github.com/az-digital/arizona-bootstrap-packagist/blob/main/.github/workflows/release.yml?rgh-link-date=2023-06-01T21%3A48%3A31Z)
> 
> So that it knows to switch versions when a 5.x release is made.

@trackleft Do you think that needs to block this PR getting merged?  Maybe we could create a follow-up issue for adding release branches to this repo?